README.md

```diff
@@ -1,12 +1,14 @@
-# Terraform Basic Sample
+# Terraform Demo Advanced
 
 A sample Terraform setup for Checkly. When applied, will provision:
-- 3 browser checks
+- 7 browser checks
 - 6 (placeholder) API checks
 - 3 groups
 - 2 alert channels
+- 1 maintenance window
+- 1 dashboard
 
-Scripts for browser checks are found in `/scripts`. With the current configuration, a browser check will be created for each script in `/scripts`.
+Browser checks are found in `/critical-user-flows`. With the current configuration, a browser check will be created for each spec in `/critical-user-flows`.
```

Comment:
@modern-sapien let's try to keep precise language here as to avoid confusion for new users - I'd say `critical-user-flows` contains _just the scripts_, not the "whole" browser check. In other words, the check itself is more than the script, and in the code config we have here roughly maps to the resource defined in the main.tf + the script in `critical-user-flows`.